frontend/src/component/common/UserAvatar/UserAvatar.tsx

```diff
@@ -19,6 +21,27 @@ const StyledAvatar = styled(Avatar)(({ theme }) => ({
     fontWeight: theme.fontWeight.bold,
 }));
 
+const StyledTooltip = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    gap: theme.spacing(2),
+}));
+
+const StyledTooltipAvatar = styled(Avatar)(({ theme }) => ({
+    width: theme.spacing(5),
+    height: theme.spacing(5),
+    margin: 'auto',
+    backgroundColor: theme.palette.secondary.light,
+    color: theme.palette.text.primary,
+    fontSize: theme.fontSizes.smallerBody,
+    fontWeight: theme.fontWeight.bold,
```

Comment:
This, presumably, is to handle cases when we put letters instead of images?

Reply:
Which, ... I'm not sure we'll ever do? Because we only show this if we have src or imageurl?

Reply:
Yep, this made me rethink my approach. I'm [now](https://github.com/Unleash/unleash/pull/9050/commits/201c15a1c0c2d12f477fb5460f0b69cf10a7861d) always showing an avatar in the tooltip with whatever content is in the original avatar, which is probably more reasonable and consistent:

![image](https://github.com/user-attachments/assets/cb8ce7a6-3c30-40bf-b4bf-957f1bf29e58)

![image](https://github.com/user-attachments/assets/74dbc826-3db7-48a7-8b3e-9f1e7f1220ee)